### DISCUSSION TITLE
Why does LocalSend does not exists as an PWA/WebApp?

### DISCUSSION THREAD

**Discussion by Z0pyrus** [localsend/localsend#1201] [Discussion (English, International)] [answered]
I think it would be nice to have, so you don't have to install it on a random pc.

**Comment by Tienisto** [comment, marked as answer]
LocalSend doesn't use WebRTC to avoid any third party servers.

A downside of that is that you need to scan the local network. This is something browsers don't allow due to privacy reasons.
